Snowflake.Data/Snowflake.Data.csproj

```diff
@@ -38,7 +34,7 @@
     <Reference Include="System.Net.Http" />
   </ItemGroup>
 
-  <ItemGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+  <ItemGroup Condition="'$(TargetFramework)' == 'net6.0'">

```

Comment:
why net 6.0 here?

Reply:
I've reverted it back, it was just for testing purposes